app/assets/javascripts/discourse/app/components/directory-table.js

```diff
@@ -2,16 +2,88 @@ import Component from "@ember/component";
 import { action } from "@ember/object";
 
 export default Component.extend({
-  classNames: ["directory-table-container"],
+  lastScrollPosition: 0,
+  ticking: false,
+  topHorizontalScrollBar: null,
+  tableContainer: null,
+
+  didRender() {
```

Comment:
`didRender` can happen on re-render, so you'd be adding all those event listeners over and over every time it re-renders right? Is there a reason you can't do it in `didInsertElement`?

Reply:
No reason, just mixed the 2 hooks up! Just swapped in `didInsertElement`